sidebars.js

```diff
@@ -75,6 +75,7 @@ module.exports = {
                     'design/state_aware_scheduling',
                     'design/scheduler_object_states',
                     'design/scheduler_core_design',
+                    'design/simple_preemptor'
```

Comment:
This should not be in between the two old/no longer maintained docs. It should be higher up the list, around the generic resource line